### DISCUSSION TITLE
Debugging error after login

### DISCUSSION THREAD

**Discussion by gvlx** [Budibase/budibase#3539] [Help] [answered]
When a 'basic' user logs in there are many things that are hidden from his view.
In my case, I have a filter on its e-mail, so as to hide all the data he shouldn't see.

He's still receiving the following error:
![error after login](https://user-images.githubusercontent.com/67311/143420466-cafe13dd-92dc-40d1-9888-db400a030567.png)

After that brief message, he has access to all his data and can do all the operations.
But it is still annoying to see that message.

How can I trace the error to find its root-cause?

Best regards,

**Comment by mjashanks** [comment, marked as answer]
Hi Gerardo.

If you 
1. Open developer tools, and click on the network tab
2. Refresh your page, whilst in your app
3. Lookout for any failed requests, and post a screenshot in here.

Would be good to know which request is failing. 

Thanks!

**Reply by gvlx** [reply]
Hi,

It's getting a 403 error on ta_users.
But I've set the Users table "Read" permissions to Public.

Any ideas?

**Reply by mjashanks** [reply]
Thats interesting - set read permission on Users to Basic.

They definitely shouldn't be public anyway, that means anyone who knows your URL can potentially see all your users.

**Reply by gvlx** [reply]
Just did it for debug purposes, back into private "Basic" access mode ;)

With screen capture:
![2021-11-25-11:28:44-Seleção-001](https://user-images.githubusercontent.com/67311/143434082-feecca2e-416f-4e8a-bd67-9cc769944ca2.png)

And logged user has the role "Basic":
![2021-11-25-11:33:01-Seleção-001](https://user-images.githubusercontent.com/67311/143434361-259cff60-b904-4c07-8e91-e9b3546e7d5f.png)

**Reply by mjashanks** [reply]
Hmm - did you change the access to `Users` at any point?

Is it possible that you changed the access level to `Users` then published the app, and have not re-published since changing back to `basic` ?

**Reply by gvlx** [reply]
Hi,

- Published
- removed user and recreated it again

No more errors.
I'm going to check other users to see if the error disappeared.

Thanks @mjashanks ;)

**Reply by gvlx** [reply]
Verified.
